packs/spell-effects/spell-effect-community-repair.json

```
@@ -0,0 +1,100 @@
+        },
+        "duration": {
+            "expiry": "turn-start",
+            "sustained": false,
+            "unit": "days",
+            "value": 7
+        },
+        "level": {
+            "value": 1
+        },
+        "rules": [
+            {
+                "choices": [
+                    {
+                        "label": "PF2E.Check.Result.Degree.Check.criticalSuccess",
+                        "value": 10
+                    },
+                    {
+                        "label": "PF2E.Check.Result.Degree.Check.criticalFailure",
+                        "value": -10
+                    }
+                ],
+                "flag": "outcome",
+                "key": "ChoiceSet",
+                "prompt": "PF2E.SpecificRule.Prompt.DegreeOfSuccess",
+                "rollOption": "community-repair-speed"
+            },
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "community-repair-proximity"
+                ],
+                "selector": "speed",
+                "type": "status",
+                "value": "@item.flags.pf2e.rulesSelections.outcome"
+            },
+            {
+                "domain": "all",
+                "key": "RollOption",
+                "label": "Within 60 feet of a Community Repair Ritual",
+                "option": "community-repair-proximity",
+                "toggleable": true
+            },
+            {
+                "key": "FlatModifier",
+                "predicate": [
+                    "emotion",
+                    "community-repair-proximity",
+                    "community-repair-speed:10"
+                ],
+                "selector": "will",
+                "type": "status",
+                "value": 1
+            },
+            {
+                "key": "Note",
+                "outcome": [],
+                "predicate": [
+                    "community-repair-proximity",
+                    "community-repair-speed:-10"
+                ],
+                "selector": "attack",
+                "text": "You cannot benefit from emotion effects.",
+                "title": "Community Repair"
+            },
```

```suggestion
            {
                "key": "Note",
                "predicate": [
                    "terrain:community-repair",
                    "community-repair-speed:-10"
                ],
                "selector": [
                    "attack-roll",
                    "skill-check",
                    "perception",
                    "saving-throw"
                ],
                "text": "You cannot benefit from emotion effects.",
                "title": "{item|name}"
            }
```
We can localize the note string later but at least it's easy enough to remove one of the note rules now.